modules/src/test/java/org/archive/modules/extractor/ExtractorHTMLTest.java

```diff
@@ -329,7 +402,6 @@ public void testScriptTagWritingScriptType() throws URIException {
     public void testOutLinksWithBaseHref() throws URIException {
         CrawlURI puri = new CrawlURI(UURIFactory
                 .getInstance("http://www.example.com/abc/index.html"));
-        puri.setBaseURI(puri.getUURI());
```

Comment:
This was the change needed because BaseURI can now only be set once in ExtractorHTML